src/frequenz/sdk/config/_config_managing.py

```diff
@@ -19,21 +20,57 @@
 
 
 class ConfigManagingActor(Actor):
-    """An actor that monitors a TOML configuration file for updates.
-
-    When the file is updated, the new configuration is sent, as a [`dict`][], to the
-    `output` sender.
-
-    When the actor is started, if a configuration file already exists, then it will be
-    read and sent to the `output` sender before the actor starts monitoring the file
-    for updates. This way users can rely on the actor to do the initial configuration
-    reading too.
+    """An actor that monitors a TOML configuration files for updates.
+
+    When the actor is started the configuration files will be read and sent to the
+    output sender. Then the actor will start monitoring the files for updates.
+
+    If no configuration file could be read, the actor will raise a exception.
+
+    The configuration files are read in the order of the paths, so the last path will
```

Comment:
I'd suggest to add a brief note to mention that an update on any config file of the monitored paths will make the ConfigManager to read/load all the files in the monitored paths.

Reply:
updated (added the clarification in another previous paragraph)